modules/auth/repo_branch_form.go

```diff
@@ -5,8 +5,8 @@
 package auth
 
 import (
-	"github.com/go-macaron/binding"
-	macaron "gopkg.in/macaron.v1"
+	"gitea.com/macaron/binding"
+	macaron "gitea.com/macaron/macaron"
```

Comment:
We should replace all `macaron "gitea.com/macaron/macaron"` to `"gitea.com/macaron/macaron"` because the alias is unnecessary.